### DISCUSSION TITLE
Forward domain with a specific port or specific resource path

### DISCUSSION THREAD

**Discussion by diasdmhub** [NginxProxyManager/nginx-proxy-manager#3335] [Q&A]
I usually expose my local services with port forwarding in my router.
The router also updates my single DDNS.

With NPM, I wish to expose the internal services with HTTPS, but my current DDNS provider does not offer subdomains.
So I was thinking of using NPM to forward my domain using either ports or resource paths.

Using ports, it would look something like this:
```
domain.pub:443 --> 1.domain.local
domain.pub:553 --> 2.domain.local
domain.pub:663 --> 3.domain.local
```

Or using paths:
```
domain.pub/1 --> 1.domain.local
domain.pub/2 --> 2.domain.local
domain.pub/3 --> 3.domain.local
```

At the moment, I can only forward the domain to a single internal host.
It does not seem possible to add a source domain with a specific port or specific resource path.

Is this possible with NGINX Proxy Manager?

**Comment by Arthurdw** [comment]
Hey there, did you find any way to achieve this?

**Reply by diasdmhub** [reply]
No, I did not. Not with NPM.
I was looking at Caddy. Maybe it will be easier for me.
